src/njs_object.c

```diff
@@ -2596,14 +2596,6 @@ njs_object_prototype_has_own_property(njs_vm_t *vm, njs_value_t *args,
     njs_value_t           *value, *property, lvalue;
     njs_property_query_t  pq;
 
-    value = njs_argument(args, 0);
```

Comment:
Suggested commit log change

`Making sure conform ECMAScript 2026 20.1.3.2, 20.1.3.3 and 20.1.3.4.`
->
`Fixed Object prototype methods according to the according to the spec.`